docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
+===============================
+
+This example shows the coriolis parameters for the area encompassing the UK.
+
+It creates and plots a map showing the coriolis parameters, overlaid on top of
+the UK coastline. The code also creates a custom cube to store the data in and
+to plot it from.
+
+"""
+
+import cartopy.crs as ccrs
+
+import iris
+
+from iris.coord_systems import GeogCS
+
+import iris.quickplot as qplt
+
+import matplotlib.pyplot as plt
+
+import numpy as np
+
+
+def calculate_coriolis_parameter(latitude):
+    """
+    This function computes the coriolis parameters for the points in an array.
+
+    It takes a 2D latitude array as its input.
+
+    It returns a 2D array for coriolis parameters.
+    """
+    omega = 7.29E-5
+    return 2. * omega * np.sin(latitude)
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 50
+    latitudes = np.linspace(49.0, 60.0, resolution)
+    longitudes = np.linspace(-11.0, 3.0, resolution)
+    lat_calc = np.deg2rad(latitudes)
+    lon_calc = np.deg2rad(longitudes)
+    lat1, long1 = np.meshgrid(lat_calc, lon_calc)
```

Comment:
@ajdawson the benefit of separating out the computation of the coriolis parameter was that it gave us a nice, small and single purpose function whose sole purpose was performing the computation. This meant we could separate the data computation out from the rest of the code, which may aid the code's legibility.

I agree that performing the `deg2rad` conversion inline is a good idea though.